tools: unified file structure, grammar, additional descriptions, and styling (#166)
lukso-network/docs: docs/tools/lsp-factoryjs/deployment/04-fungible-token.md

---
sidebar_position: 4
title: Fungible Token (LSP7)
---

# Fungible Token (LSP7)

[LSP7](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-7-DigitalAsset.md) is a standard for either fungible or non-fungible tokens based on the ERC20 token standard.

## Deploying an LSP7 Digital Asset

> To create an LSP7 Fungible token, use the `LSP7DigitalAsset.deploy()` method and set the `isNFT` parameter to `false`. The code sequence will deploy an LSP7 smart contract with many tokens that can be fractionalized.

Reviewer: ```suggestion
To create an LSP7 Fungible token, use the `LSP7DigitalAsset.deploy()` method and set the `isNFT` parameter to `false`. This will deploy an LSP7 smart contract which can have many tokens that can be fractionalized.
```

> To create an LSP7 Fungible token, use the `LSP7DigitalAsset.deploy()` method and set the `isNFT` parameter to `false`. The code sequence will deploy an LSP7 smart contract with many tokens that can be fractionalized.

Reviewer: ```suggestion
To create an LSP7 Fungible token, use the `LSP7DigitalAsset.deploy()` method and set the `isNFT` parameter to `false`. This will deploy an LSP7 smart contract which can have many tokens that can be fractionalized.
```
